include/qpid/dispatch/router_core.h

```diff
@@ -678,6 +680,8 @@ typedef uint64_t (*qdr_link_deliver_t)   (void *context, qdr_link_t *link, qdr_d
 typedef void (*qdr_delivery_update_t)    (void *context, qdr_delivery_t *dlv, uint64_t disp, bool settled);
 typedef void (*qdr_connection_close_t)   (void *context, qdr_connection_t *conn, qdr_error_t *error);
 
+typedef qdr_address_t * (*qdr_edge_conn_addr_t) (void *context);
```

Comment:
Why is this in the interface to the router_core?  What component external to the router_core will ever use this?